libs/hdf-converters/src/utils/attestations.ts

```diff
@@ -181,9 +229,15 @@ function getFirstPath(
     throw new Error(
       `Attestation is missing one of these paths: ${paths.join(', ')}`
     );
-  } else {
-    return _.get(object, paths[index]) as string;
   }
+  const stringOrDate = _.get(object, paths[index]);
+  if (_.isString(stringOrDate)) {
+    return stringOrDate;
+  }
+  //return _.get(object, paths[index]);
+  // Do we care about time?
+  console.log('stringorDate: ' + stringOrDate);
+  return `${stringOrDate.getFullYear()}-${String(stringOrDate.getMonth() + 1).padStart(2, '0')}-${String(stringOrDate.getDate()).padStart(2, '0')}`;
```

Comment:
This change is not clear to me. Can you explain more about this section?

Reply:
Excel stores dates as Date objects and they don't format nicely when converting to a string using sheetjs. This returns a string version of the 'updated' field with the padding needing to have it resemble what the data would otherwise look like coming from normal attestation files.